main/acle.rst

```diff
@@ -404,6 +408,182 @@ Scalable Vector Extensions (SVE)
+| .. code:: c                                                                    |
+|                                                                                |
+|      svuint64_t svset_neonq[_u64](svuint64_t vec, uint64x2_t subvec)           |
++--------------------------------------------------------------------------------+
+| .. code:: c                                                                    |
+|                                                                                |
+|      svfloat16_t svset_neonq[_f16](svfloat16_t vec, float16x8_t subvec)        |
++--------------------------------------------------------------------------------+
+| .. code:: c                                                                    |
+|                                                                                |
+|      svfloat32_t svset_neonq[_f32](svfloat32_t vec, float32x4_t subvec)        |
++--------------------------------------------------------------------------------+
+| .. code:: c                                                                    |
+|                                                                                |
+|      svfloat64_t svset_neonq[_f64](svfloat64_t vec, float64x2_t subvec)        |
++--------------------------------------------------------------------------------+
+
+``svget_neonq``
+~~~~~~~~~~~~~~~
+
+These intrinsics return a NEON vector from the first 128 bit subvector of
+the SVE vector ``vec``.
+
++--------------------------------------------------------------+
+| **Instances**                                                |
++==============================================================+
+| .. code:: c                                                  |
+|                                                              |
+|      int8x16_t svget_neonq[_s8](svint8_t vec)                |
++--------------------------------------------------------------+
+| .. code:: c                                                  |
+|                                                              |
+|      int16x8_t svget_neonq[_s16](svint16_t vec)              |
++--------------------------------------------------------------+
+| .. code:: c                                                  |
+|                                                              |
+|      int32x4_t svget_neonq[_s32](svint32_t vec)              |
++--------------------------------------------------------------+
+| .. code:: c                                                  |
+|                                                              |
+|      int64x2_t svget_neonq[_s64](svint64_t vec)              |
++--------------------------------------------------------------+
+| .. code:: c                                                  |
+|                                                              |
+|      uint8x16_t svget_neonq[_s8](svuint8_t vec)              |
```

Comment:
Typo here and below:  `s` should be `u`.

Reply:
Thanks :)